manywheel/build_rocm.sh

```diff
@@ -97,30 +119,40 @@ ROCM_SO_FILES=(
     "libroctx64.so"
 )
 
+# Adjust list based on ROCm version
 if [[ $ROCM_INT -ge 50600 ]]; then
```

Comment:
Please take a look at upstream build_rocm.sh to remove the conditions for old ROCm versions